label_studio/tasks/models.py

```diff
@@ -247,6 +249,25 @@ def has_lock(self, user=None):
     def num_locks(self):
         return self.locks.filter(expire_at__gt=now()).count()
 
+    def overlap_with_agreement_threshold(self, num, num_locks):
+        try:
+            from stats.models import get_task_agreement
+        except (ModuleNotFoundError, ImportError):
+            pass
```

Comment:
This should be moved within the if check below. No point in adding a failure condition for 50% of the use cases.